README.md

```diff
@@ -221,7 +223,7 @@ You can find helpful code snippets in the `examples` folder.
 imgproxy can process files from your local filesystem. To use this feature do the following:
 
 1. Set `IMGPROXY_LOCAL_FILESYSTEM_ROOT` to your images directory path.
-2. Use `local:///path/to/image.jpg` as the source image url.
+2. Use `local:///path/to/image.jpg` as the source image url. The path should be relative to the filesystem root value
```

Comment:
Sorry, I added this sentence, because I wasted half an hour trying to understand why it doesn't work for me. Turns out I set absolute path which is wrong.

Reply:
This shouldn't be a part of this PR